_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -0,0 +1,92 @@
+Introduced 2.8
+{: .label .label-purple }
+
+Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly. Use the **Dashboard** app to:
+
+- Set up, add, and adjust rules and conditions that trigger alerts and notifications.
+- Create graphs that show trends and patterns and build intuitive dashboards to stay on top of important metrics and data points in real time.
+- Monitor your alerts in one place with at-a-glance views.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/alerting-viz.png" alt="Example alerting visualization" width="800" height="800">
+
+## Getting started 
+
+Before getting started, you must have:
+
+- Installed OpenSearch and OpenSearch Dashboards version 2.8 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
+- Installed Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+
+## Configuring admin settings
+
+You can only access, create, or manage alerts for resources for which you have permissions. Access to alerting dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards privileges, and you can manage the settings in **Stack Management**. Access is enabled by default and appears as a feature in the **Stack Management** > **Advanced Settings** > **Visualizations** window. If the setting is disabled, it does not appear in this window. The setting is disabled at the cluster level through the `opensearch-dashboards.yml` file.
+
+## General requirements for alerting visualizations
+
+Alerting visualizations are displayed as time-series charts that give you a snapshot of the alert, alert status, last updated time, and reason for the alert. You can show up to 10 metrics on your chart, and each series can be shown as a line in the chart.
+
+Keep in mind the following requirements when setting up or creating alerting visualizations:
+
+- Must be a [Vizlib line chart](https://community.vizlib.com/support/solutions/articles/35000107262-vizlib-line-chart-introduction)
+- Must contain one Y-axis metric aggregation
+- Must not have non-Y-axis metric aggregation types
+- Must use date histogram aggregation type for the X-axis bucket
+- Must have X-axis on the bottom
+- Must define one X-axis aggregation bucket
+- Must have a valid time-based X-axis
+
+## Creating alerting monitors with Dashboard
+
+By default, when you begin to create the alert monitor workflow using the Dashboard app, you are presented with a menu-driven interface. This interface provides a range of options, displayed in full screen, pop-up, pull-down, or dropdown, to define the metrics to monitor, set thresholds, customize triggers that automate workflows, and generate actions when conditions are met. Currently, you can create query-level monitors only.
```

Comment:
Query-level is referred to as 'per query' in documentation. Let's adjust for consistency. https://opensearch.org/docs/latest/observing-your-data/alerting/monitors/#monitor-types